srcpkgs/xdelta3/template

```diff
@@ -1,12 +1,21 @@
 # Template file for 'xdelta3'
 pkgname=xdelta3
-version=3.0.11
+version=3.1.0
 revision=1
 build_style=gnu-configure
 makedepends="liblzma-devel"
 short_desc="Delta/differential compression tools, VCDIFF/RFC 3284 delta compression"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-license="GPL-2"
+license="GPL-2.0-only"
 homepage="http://xdelta.org"
 distfiles="https://github.com/jmacd/xdelta-devel/releases/download/v$version/$pkgname-$version.tar.gz"
-checksum=0ccc9751ceaa4d90d6b06938a4deddb481816f5d9284bd07d2e728609cb300aa
+checksum=114543336ab6cee3764e3c03202701ef79d7e5e8e4863fe64811e4d9e61884dc
+
+case "${XBPS_TARGET_MACHINE}" in
+	*-musl) broken="Several undefined references";;
```

Comment:
The message is rather unspecific. Do you have a link to a build log, can't those be fixed by patching the package?

Reply:
Got it to compile this time, I think maybe I didn't do xbps-src clean after using LDFLAGS to fix aarch64* build